llvm/lib/Target/EraVM/MCTargetDesc/EraVMMCTargetDesc.h

```diff
@@ -50,6 +51,14 @@ MCAsmBackend *createEraVMMCAsmBackend(const Target &T,
 
 std::unique_ptr<MCObjectTargetWriter> createEraVMELFObjectWriter(uint8_t OSABI);
 
+struct EraVMOpcodeInfo {
```

Comment:
Could we add that it represents the table `EraVMOpcodesList` from `EraVMOpcodes.td` and that fields description is there (or perhaps it worth to duplicate to not to lookup there)?

Reply:
Thank you! It's very helpful.